activities/member-relations/index.md

```diff
@@ -4,4 +4,74 @@ type: Index
 
 # Member relations
 
+The Foundation for Public Code is a member-owned association. Only wholly publicly owned organizations may become members. Members participate in the steering and governance of the association by voting at the general assembly. They also participate by financing the association through yearly dues.
+
 We support members in getting the most from their membership and [codebase stewardship](../codebase-stewardship), as well as building and maintaining relationships with and between members.
+
+The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership  processes may differ between the organizations we work with.
+
+## 1. Membership growth
+
+During member growth we connect with and support wholly publically-owned organizations in their path towards membership.
+
+In this phase, we aim to:
+
+* build trust in our organization and our processes
+* determine focal and contact points within the prospective member organization
+* gain insight into the member's organizational structures and how codebase stewardship can benefit them
+
+The following are examples of activities we may conduct during member acquisition:
+
+* Conduct workshops to raise awareness about
+  * Codebase stewardship
+  * Public code
+  * The Standard for Public Code
+  * Our organizational model
+* Hold meetings to exchange knowledge and support with contracting
+* Have regular calls to answer any questions and ensure continuous, open communication
+
+Member growth is finalized when the public organization signs a [membership contract](../membership-growth/membership-contract.md).
+
+## 2. Member onboarding
+
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
```

Comment:
```suggestion
During member onboarding we help new members learn our processes, and support their public code projects.
```

Reply:
"Get accustomed to" sounds passive, slow and perhaps a bit unpleasant - like growing a callous where a too-tight shoe rubs your foot. Is there a more active/proactive/positive description of this phase?